spec/classes/init_spec.rb

```diff
@@ -672,4 +672,34 @@
       end
     end
   end
+
+  describe 'with disable_monitor set' do
+    let(:facts) { { :osfamily => 'RedHat' } }
+
+    [true,'true',false,'false'].each do |value|
+      context "to #{value} as #{value.class}" do
+        let(:params) { { :disable_monitor => value } }
+
+        it { should compile.with_all_deps }
+        it { should contain_class('ntp') }
+
+        if value == true or value == 'true'
+          it { should contain_file('ntp_conf').with_content(/^disable monitor$/) }
+        else
```

Comment:
This would say that if disable_monitor is not true then it should not be in the file. If it is set to 'asdf' it should error. Suggest this

``` ruby
if value.to_s == 'true'
  it { should contain_file('ntp_conf').with_content(/^disable monitor$/) }
end
if value.to_s == 'false'
  it { should contain_file('ntp_conf').without_content(/^disable monitor$/) }
end
```